src/services/data/comments.ts

```diff
@@ -62,6 +62,51 @@ export default (client, httpAuth: HttpInstance) => ({
     ).data;
   },
 
+  /**
+   * Find By Id
+   * @param id the Id to search for
+   * @param schemaId the schema Id
+   * @param documentId the document Id
+   * @param rql an optional rql string
+   * @returns the first element found
+   */
+  async findById(
+    id: ObjectId,
+    schemaId: ObjectId,
+    documentId: ObjectId,
+    rql?: RQLString
```

Comment:
Optional parameters should go in seperate object.

Reply:
I wasn't sure if we should... as the find helpers should be kept simple.

Reply:
done!